fzf-marks.plugin.bash

```diff
@@ -36,7 +36,7 @@ if [[ -z "${FZF_MARKS_COMMAND}" ]] ; then
     MINIMUM_VERSION=16001
 
     if [[ $FZF_VERSION -gt $MINIMUM_VERSION ]]; then
-        FZF_MARKS_COMMAND="fzf --height 40% --reverse --header='ctrl-y:jump, ctrl-t:toggle, ctrl-d:delete, ctrl-k:paste'"
+        FZF_MARKS_COMMAND="fzf --height 40% --reverse --header=\"\$_fzm_keymap_description\""
```

Comment:
One thing that I want to ask is whether there is a reason to include the `--header` option in `FZF_MARKS_COMMAND`. Why don't we specify the `--header` option in the caller side?

Reply:
Could you elaborate? I am not sure that I understand the question...

Reply:
Sorry for my unclear question. Currently, the option `--header="$_fzm_keymap_description"` is specified in `FZF_MARKS_COMMAND`, and the actual contents of the header are specified at the caller sides (`fzm`, `jump`, `dmark`, and `pmark`), whose structure can be illustrated by the following code:

```bash
# Initialize
FZF_MARKS_COMMAND='fzf --height 40% --reverse --header="$_fzm_keymap_description"'

# Caller side
local _fzm_keymap_description='<HEADER_CONTENT>'
eval "$FZF_MARKS_COMMAND" --more-options...
```

I thought it is cleaner to directly specify the header option at the caller side in the following way:

```bash
# Initialize
FZF_MARKS_COMMAND='fzf --height 40% --reverse'

# Caller side
eval "$FZF_MARKS_COMMAND" --header='<HEADER_CONTENT>' --more-options...
```

But after I asked the question, I checked the minimum version for the `--header` option, and it was `0.10.5` (https://github.com/junegunn/fzf/commit/2022a3a on 2015-09-15) which is later than the project start of `fzf-marks` (initial commit 2015-06-04). Maybe this is the reason that `--header` is included in `FZF_MARKS_COMMAND`? (Another option included in `FZF_MARKS_COMMAND`, `--height` which is supported from `0.16.00` (https://github.com/junegunn/fzf/commit/1448d631 2017-01-08), is used only when `[[ $FZF_VERSION -gt $MINIMUM_VERSION ]]`.)

Thank you

Reply:
Hi @akinomyoga ,

I understand what you mean now, and I agree that your proposed solution seems cleaner. In the pull request that introduced `--header`, the header was hardcoded, so I don't think there was a version issue.

Reply:
Oh, I see! Thank you for pointing to the background of introducing `--header`! Then, I'll make another PR for this refactoring. Besides this one, I also have two other small fixes for which I'll open PRs soon. Thank you again!